qutip/qip/device/processor.py

```diff
@@ -794,9 +847,37 @@ def plot_pulses(self, title=None, figsize=(12, 6), dpi=None):
                 ax.spines['bottom'].set_visible(False)
                 ax.spines['left'].set_visible(False)
                 ax.set_yticks([])
-                ax.set_ylabel(label,  rotation=0)
+                ax.set_ylabel(label, rotation=0)
                 pulse_ind += 1
-        if title is not None:
-            ax.set_title(title)
+                if i == 0 and j == 0 and title is not None:
+                    ax.set_title(title)
```

Comment:
Could remain outside the loop, but probably just personal preference:
```python
for i, label_group in ...:
    [plotting stuff]
if axis and title:
    axis[0].set_title(title)
```
Is there a reason it only sets the title for one axis?  I'm sure there is, just I don't see it immediately.

Reply:
Indeed this is on purpose, n axes are arranged in a column and they need only one title. It can also be down with `fig` I guess but I think set the title for `axis[0]` is safer for the position of the title.